Applications/spire/include/spire/charting/chart_view.hpp

```diff
@@ -50,6 +50,10 @@ namespace Spire {
       //! Removes the crosshair from the chart.
       void reset_crosshair();
 
+      //! Returns a pair (top left, bottom right) of ChartPoints
```

Comment:
Use ```std::tuple```, ```std::pair``` is obsolete.

Reply:
So how do tuples work with structured bindings? A pair guarantees that only two values are returned, but a tuple can be arbitrarily long. Is there a way to make its length explicit?

Reply:
```std::tuple``` works in a similar way as ```std::pair```. To access its members you use ```std::get<I>``` where ```I``` is an integer representing the 0-based index of the element you want to access:

https://en.cppreference.com/w/cpp/utility/tuple